adapter/internal/oasparser/envoyconf/constants.go

```diff
@@ -79,11 +79,13 @@ const (
 const (
 	healthPath  string = "/health"
 	testKeyPath string = "/testkey"
+	readyPath   string = "/ready"
```

Comment:
Please modify the k8s artefacts accordingly as well

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/2101/commits/b1549c07886f6a498d0323b293f922bc4fe2b3d8